src/Hazelcast.Net/DistributedObjects/Impl/HRingBuffer.cs

```diff
@@ -45,9 +44,12 @@ internal class HRingBuffer<TItem> : DistributedObjectBase, IHRingBuffer<TItem>
         public HRingBuffer(string name, DistributedObjectFactory factory, Cluster cluster, int maxBatchSize, ISerializationService serializationService, ILoggerFactory loggerFactory)
             : base(HRingBuffer.ServiceName, name, factory, cluster, serializationService, loggerFactory)
         {
-            _maxBatchSize = maxBatchSize;
+            MaxBatchSize = maxBatchSize;
```

Comment:
This is not a configurable option parameter. It's an internal const value.
On Java:
```
    /**
     * The maximum number of items that can be retrieved in 1 go using the
     * {@link #readManyAsync(long, int, int, IFunction)} method.
     */
    public static final int MAX_BATCH_SIZE = 1000;
```

Reply:
yes... guess I don't like magic constants and just converted it ;) will revert